functions/routes/galaxpay/webhooks.js

```diff
@@ -9,372 +27,136 @@ exports.post = ({ appSdk, admin }, req, res) => {
   // POST transaction.updateStatus update Transation status
   // POST subscription.addTransaction add transation in subscription
 
-  const galaxpayHook = req.body
-  const type = galaxpayHook.event
-  const GalaxPaySubscription = galaxpayHook.Subscription
-  const GalaxPaySubscriptionQuantity = GalaxPaySubscription.quantity
-  const subscriptionId = GalaxPaySubscription.myId
-  const GalaxPayTransaction = galaxpayHook.Transaction
-  const GalaxPayTransactionValue = GalaxPayTransaction.value / 100
-
-  console.log('> Galaxy WebHook ', type, ' Body Webhook ', JSON.stringify(galaxpayHook), ' quantity: ', GalaxPaySubscriptionQuantity, ' status:', GalaxPayTransaction.status, ' <')
   const collectionSubscription = admin.firestore().collection('subscriptions')
 
-  const checkStatus = (financialStatus, GalaxPayTransaction) => {
-    if (financialStatus.current === parseStatus(GalaxPayTransaction.status)) {
-      return true
-    } else if ((financialStatus.current === 'paid' || financialStatus.current === 'authorized') && parseStatus(GalaxPayTransaction.status) !== 'refunded') {
-      return true
+  const { body } = req
+  const type = body.event
+  // const { Transaction, Subscription } = body
+  const webhookTransaction = body.Transaction
+  const webhookSubscription = body.Subscription
+  const galaxpaySubscriptionId = webhookSubscription.myId
+  const metadataStoreId = webhookSubscription.ExtraFields.find(metadata => metadata.tagName === 'store_id')
```

Comment:
```suggestion
  const type = body?.event
  if (!type) {
    return res.sendStatus(400)
  }
  // const { Transaction, Subscription } = body
  const webhookTransaction = body.Transaction
  const webhookSubscription = body.Subscription
  const galaxpaySubscriptionId = webhookSubscription?.myId
  if (!galaxpaySubscriptionId) {
    return res.sendStatus(400)
  }
  const metadataStoreId = webhookSubscription.ExtraFields?.find(metadata => metadata.tagName === 'store_id')
```

Só pra evitar (pelo menos um pouco) fatal com corpo inválido (a.k.a. se uma pessoa ou bot enviar um POST qualquer nesse endpoint)